data/tlg0067/tlg001/tlg0067.tlg001.1st1K-grc1.xml

```diff
@@ -85,14 +85,14 @@ Architecture.</p>
 <refsDecl n="CTS">
 
 <cRefPattern n="paragraph" matchPattern="(.+).(.+).(.+)"
```

Comment:
You should change your @n attributes here to reflect what the @subtype is on the elements (e.g., div or p).